demo_nodes_cpp/src/parameters/parameter_blackboard.cpp

```diff
@@ -29,12 +29,9 @@ class ParameterBlackboard : public rclcpp::Node
 public:
   DEMO_NODES_CPP_PUBLIC
   explicit ParameterBlackboard(
-    const rclcpp::NodeOptions & options = (
-      rclcpp::NodeOptions()
-      .allow_undeclared_parameters(true)
-      .automatically_declare_parameters_from_overrides(true)
-  ))
```

Comment:
The default value was not being used, right?
IMO, it's ok to `force allow_undeclared_parameters` `automatically_declare_parameters_from_overrides` to `true`, because if not the example doesn't have sense.

Reply:
these are default arguments, so not used with current implementation.